front/app/components/admin/PostManager/components/PostTable/Row/selectors/ProjectSelector.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 
-import { Label } from 'semantic-ui-react';
+import { Label } from '@citizenlab/cl2-component-library';
 import styled from 'styled-components';
 
 import useProjectById from 'api/projects/useProjectById';
 
 import T from 'components/T';
 
-const LabelText = styled.span`
-  font-weight: 600;
-`;
-
 const StyledLabel = styled(Label)`
+  display: inline;
   white-space: nowrap;
   -webkit-line-clamp: 1;
   -webkit-box-orient: vertical;
   overflow: hidden;
   text-overflow: ellipsis;
-  max-width: 175px;
+  color: ${({ theme }) => theme.colors.teal};
```

Comment:
Feels like a lot of custom styling for a component that should be relatively standard. Is there a way to reduce? Not blocking, if not.